scripts/mobility-database-harvester/harvest_latest_versions.py

```diff
@@ -145,23 +51,29 @@ def save_content_to_file(content, data_path, filename):
         json.dump(content, f)
 
 
-def load_content(data_path, filename):
-    file_path = path.join(data_path, filename)
-    with open(file_path, "r") as f:
-        content = json.load(f)
-    return content
+def harvest_latest_versions():
+    catalogs = pd.read_csv(CATALOGS_CSV)
+    latest_versions = {}
+
+    latest_urls = catalogs[LATEST].loc[catalogs[DATA_TYPE] == GTFS]
+    for index, value in latest_urls.items():
+        latest_url = value
+        source_file_name = latest_url.replace(URL_PREFIX, "").replace(URL_SUFFIX, "")
+        latest_versions[source_file_name] = latest_url
+
+    return latest_versions
 
 
 def apply_github_matrix_formatting(json_data):
```

Comment:
I'm not sure json_data is an accurate or useful variable name here.  It's just a dictionary of name => url, yeah?

Reply:
Right. I changed it to `latest_urls`.